chess-AI/train.py

```diff
@@ -215,8 +215,9 @@ def train_on_dataset(dataset, nnet, options, show_dash=False):
     save_model(nnet, options.save_path, options.num_saved_models, options.overwrite)
 
 
-def train_on_stockfish(nnet, elo, depth, dataset_path, options, show_dash=False):
-    stockfish = StockfishTrain(elo, depth)
+def train_on_stockfish(nnet, dataset_path, options, show_dash=False):
```

Comment:
I realize this was introduced in my pr, so sorry for that, but we shouldn't use `options` as the parameter here since that's the name of the module. Can you pls change `options` to `sf_opt`?